src/apps/run/main.js

```diff
@@ -15,19 +16,28 @@ let inputs = new client.Input(document.getElementById("juttle-input-groups"));
 let currentBundle;
 let parsed = url.parse(window.location.href, true);
 
-getBundle(parsed.query.path)
-.then((res) => {
-    currentBundle = res.bundle;
-    return client.describe(currentBundle);
-})
-.then((desc) => {
-    // if we have no inputs go ahead and run
-    if (desc.inputs.length === 0) {
-        view.run(currentBundle);
-    } else {
-        inputs.render(currentBundle);
-    }
-})
+let initBundle = (bundle) => {
+    currentBundle = bundle;
+    client.describe(bundle)
+    .then((desc) => {
+        // if we have no inputs go ahead and run
+        if (desc.inputs.length === 0) {
+            view.run(bundle);
+        } else {
+            inputs.render(bundle);
+        }
+    });
+};
+
+if (parsed.query.path) {
+    client.describe(parsed.query.path)
+    .then(bundle => initBundle(bundle));
+} else if (parsed.query.rendezvous) {
+    let rendezvousUrl = `ws://${outriggerHost}/rendezvous/${parsed.query.rendezvous}`
```

Comment:
something like 
```
RendezvousSocket({
   host: outriggerHost,
   topic: parsed.query.rendezvous
})
```
the only thing is that you might also want to have secured socket so you would also have to support an attribute `secure` that makes it `wss://`